prog/probe.go

```diff
@@ -40,7 +40,10 @@ const (
 	defaultServiceHost = "https://cloud.weave.works:443"
 )
 
-var pluginAPIVersion = "1"
+var (
+	pluginAPIVersion = "1"
+	dockerEndpoint   = "unix:///var/run/docker.sock"
```

Comment:
> to handle docker-machine / minikube cases?

However, this won't be the usecase since scope also needs to live in the machine being monitored (it needs `/proc` and conntrack for instance)